src/libsyntax/ext/expand.rs

```diff
@@ -787,8 +787,6 @@ fn decorate(a: Annotatable, fld: &mut MacroExpander) -> SmallVector<Annotatable>
     let mut decorator_items = SmallVector::zero();
     let mut new_attrs = Vec::new();
     expand_decorators(a.clone(), fld, &mut decorator_items, &mut new_attrs);
-    let decorator_items =
-        decorator_items.into_iter().flat_map(|a| expand_annotatable(a, fld)).collect();
```

Comment:
`decorator_items` are already get expanded in `expand_decorators`.
I mistakenly added this in #34010. Since expansion in idempotent, it had no effect except for regressing expansion performance.